x-pack/plugin/eql/src/test/resources/queries-supported.eql

```diff
@@ -687,3 +687,17 @@ process where fake_field != "*"
 
 process where not (fake_field : "*")
 | head 4;
+
+
+// eventType values
+"123" where process_name == "svchost.exe";
+
+"!#$%^&!#$\"123*&^%$()" where process_name == "svchost.exe";
+
+"""""!#$%^&!""#$\"123*""&^%$()""""" where process_name == "svchost.exe";
+
+sequence
+  ["!#$134" where serial_event_id<3] by unique_pid1
+  ["""!#@$%12345)(*&%^%#$@""" where true] by unique_ppid2
+| tail 1
+;
```

Comment:
Could you please add one with space with dots as well just to consistent?
Otherwise LGTM